description.md

```diff
@@ -543,26 +543,26 @@ Exams aim to measure comprehension, critical thinking, and problem-solving skill
 
 Examinations can take various forms, including:
 
-* **Written Exams** consist of short answers, essays, or problems to solve.
+- **Written Exams** consist of short answers, essays, or problems to solve.
   They are versatile and are fit various subjects.
   Written exams build on what you've learned earlier.
   Written exams build upon previous learning, but they aren't adaptive and don't personalize the experience for each learner.
   Additionally, they might not be the best choice for assessing complex situations, often leaning towards straightforward questions.
 
-* **Multiple-Choice Exams** involve choosing the correct answer from a set of options, essentially being a compilation of [quizzes](#quizzes).
+- **Multiple-Choice Exams** involve choosing the correct answer from a set of options, essentially being a compilation of [quizzes](#quizzes).
```

Comment:
```suggestion
- **Multiple-Choice Exams** involve choosing the correct answer from a set of options, essentially being a compilation of ["Quizzes"](#quizzes).
```
We talked about this on discord, but I think it ruins the flow of reading.

Reply:
@Alex-deVis , see the new content. I preferred to use "Quizzes" only when wording it like `the "Quizzes" section`.